client_test.go

```diff
@@ -1443,7 +1443,7 @@ func TestClient_mtlsNetRPCClient(t *testing.T) {
 }
 
 func TestClient_logger(t *testing.T) {
-	t.Run("net/rpc", func(t *testing.T) { testClient_logger(t, "netrpc") })
+	t.Run("netrpc", func(t *testing.T) { testClient_logger(t, "netrpc") })
```

Comment:
This change is solely for test ergonomics, the extra `/` made it behave as a sub-sub-test.